pkg/resource/logging-config/alloy/logging-config.alloy.yaml.template

```
@@ -58,3 +61,60 @@ alloy:
           path: /run/log/journal
       - name: alloy-tmp
         emptyDir: {}
+podLogs:
+{{- if .IsWorkloadCluster }}
+- name: namespace-default
+  namespace: giantswarm
```

```suggestion
  namespace: giantswarm
```

Maybe this should bé in thé kube-system.namespace? Giantswarm namespace will soon disappear on WCs

---

Ok that makes sense, I'll move PodLogs into kube-system namespace

---

> Giantswarm namespace will soon disappear on WCs

Really? I didn't know this :open_mouth: are there some docs about it?

---

Nope but it's coming with the removal of the chart-operator on WCs

---

The goal is to not have any giant swarm code on WCs

---

PodLogs are now created in kube-system namespace